tests/test_client.py

```diff
@@ -37,7 +37,7 @@ def slug(request, cl_admin):
 
     def remove_by_slug():
         try:
-            cl_admin.delete_project(slug)
+            cl_admin.delete_project(slug, enable_undo=False)
```

Comment:
oho is it that?

Reply:
I *think* so, I also had to manually restore the project from the 'trash'. I'm not sure why. We shall see if you tests pass too.

Reply:
what is strange is how it is the same project key for all the commits.

Reply:
hmm it still not using the GH prefix, but  I think this is because it is not a default env var

Reply:
trying once more to see if I can get it to pick up the env vars, otherwise I think we can say we have fixed the tests for now

Reply:
So now it is correctly picking up the env vars:
see the CI prefixed proj name: https://github.com/pycontribs/jira/actions/runs/7338468278/job/19981037988?pr=1786#step:7:457

But note how we are getting a warning regarding the project key being >10 chars
```
  E           	response text = {"errorMessages":[],"errors":{"projectKey":"The project key must not exceed 10 characters in length."}}
```

Reply:
so looks like this has been fixed now, using the correct prefix, but the projects are not fully deleted from the Cloud instance. Might be worth looking into that as I guess the instance will need periodic purging otherwise.

Reply:
I've implemented a specific garbage collection handler to truly clear out any dangling projects.